Integrations/integration-ServiceNow.yml

```diff
@@ -27,8 +27,7 @@ configuration:
   defaultvalue: "false"
   type: 8
   required: false
-- display: Default ticket type of commands. Can be incident, sc_request, problem or
-    change_reuquest. Commands also take ticket_type argument
+- display: Default ticket type for running ticket commands and fetching incidents
```

Comment:
I understand it was misleading before, but still need to direct to how the possible option can be retrieved